webpack/assets/javascripts/react_app/common/__snapshots__/withReactRoutes.test.js.snap

```diff
@@ -0,0 +1,32 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`withReactRoutes should render and pass props: should render and pass props 1`] = `
+<Router
+  history={
```

Comment:
I afraid the interface of the `history` object is going to change between versions.
Can we mock a nicer object instead?